static/js/biomedical/landing/header_and_search_section.tsx

```diff
@@ -47,6 +47,9 @@ const HeadingContainer = styled(ContentContainer)`
 const StyledText = styled.div`
   flex-grow: 1;
 
+  a {
+    color: #146c2e;
```

Comment:
Nit: This color looks like it's used a few places in biomedical. Maybe extract it to `CSS_THEME`?

Reply:
good callout, moved to CSS_THEME